### ISSUE TITLE
Start error

### ISSUE THREAD

**Issue by andris155** [games647/ScoreboardStats#139]
```
[10:30:06] [Server thread/INFO]: [ScoreboardStats] Enabling ScoreboardStats v0.9.14
[10:30:06] [Server thread/INFO]: [ScoreboardStats] Registered replacers: [BungeeCordVariables, BukkitGlobalVariables, PlayerPingVariable, GeneralVariables, VaultVariables, BukkitVariables]
[10:30:06] [Server thread/ERROR]: ebean.properties not found
[10:30:06] [Server thread/ERROR]: [ScoreboardStats] null
java.lang.NoSuchFieldException: annotations
	at java.lang.Class.getDeclaredField(Class.java:2070) ~[?:1.8.0_101]
	at com.github.games647.scoreboardstats.pvpstats.DatabaseConfiguration.setTablePrefix(DatabaseConfiguration.java:191) [ScoreboardStats.jar:?]
	at com.github.games647.scoreboardstats.pvpstats.DatabaseConfiguration.getSqlConfig(DatabaseConfiguration.java:121) [ScoreboardStats.jar:?]
	at com.github.games647.scoreboardstats.pvpstats.DatabaseConfiguration.loadConfiguration(DatabaseConfiguration.java:75) [ScoreboardStats.jar:?]
	at com.github.games647.scoreboardstats.pvpstats.Database.setupDatabase(Database.java:194) [ScoreboardStats.jar:?]
	at com.github.games647.scoreboardstats.ScoreboardStats.onEnable(ScoreboardStats.java:141) [ScoreboardStats.jar:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:292) [patched_1.10.2.jar:git-Paper-887]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:316) [patched_1.10.2.jar:git-Paper-887]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:407) [patched_1.10.2.jar:git-Paper-887]
	at org.bukkit.craftbukkit.v1_10_R1.CraftServer.enablePlugin(CraftServer.java:375) [patched_1.10.2.jar:git-Paper-887]
	at org.bukkit.craftbukkit.v1_10_R1.CraftServer.enablePlugins(CraftServer.java:325) [patched_1.10.2.jar:git-Paper-887]
	at net.minecraft.server.v1_10_R1.MinecraftServer.t(MinecraftServer.java:418) [patched_1.10.2.jar:git-Paper-887]
	at net.minecraft.server.v1_10_R1.MinecraftServer.l(MinecraftServer.java:383) [patched_1.10.2.jar:git-Paper-887]
	at net.minecraft.server.v1_10_R1.MinecraftServer.a(MinecraftServer.java:329) [patched_1.10.2.jar:git-Paper-887]
	at net.minecraft.server.v1_10_R1.DedicatedServer.init(DedicatedServer.java:275) [patched_1.10.2.jar:git-Paper-887]
	at net.minecraft.server.v1_10_R1.MinecraftServer.run(MinecraftServer.java:584) [patched_1.10.2.jar:git-Paper-887]
	at java.lang.Thread.run(Thread.java:745) [?:1.8.0_101]
[10:30:06] [Server thread/INFO]: Using [com.github.games647.scoreboardstats.pvpstats.PathReader] to read the searchable class path
[10:30:06] [Server thread/INFO]: DataSourcePool [ScoreboardStats] autoCommit[false] transIsolation[SERIALIZABLE] min[2] max[20]
[10:30:06] [Server thread/INFO]: SubClassFactory parent ClassLoader [com.github.games647.scoreboardstats.ReloadFixLoader]
[10:30:07] [Server thread/INFO]: Entities enhanced[0] subclassed[1]
[10:30:07] [Server thread/INFO]: runScript
[10:30:07] [Server thread/INFO]: executing 1 of 1 create table IF NOT EXISTS player_stats ( id                        integer prim...
[10:30:07] [Server thread/INFO]: ... end of script
```

**Comment by games647** [comment]
Fixed with https://github.com/games647/ScoreboardStats/commit/e8cb5a2767666cac81ac6fd5f757afb86b22d785

**Comment by Someone-NEW** [comment]
What here is fixed any link with the fixed one? i cant find any thanks

**Comment by Someone-NEW** [comment]
Ok can you explain me who can find that "Fixed with e8cb5a2" jar file?? remember we are not experts with this jobs and java and making jars etc. so i just ask who is this with fixed file? Thank you
